controller/users.go

```diff
@@ -167,6 +167,15 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 			keycloakUserProfile.Username = updatedUserName
 		}
 
+		updatedRegistratedCompleted := ctx.Payload.Data.Attributes.RegistrationCompleted
+		if updatedRegistratedCompleted != nil {
+			if *updatedRegistratedCompleted == false {
+				jerrors, _ := jsonapi.ErrorToJSONAPIErrors(goa.ErrInvalidRequest(fmt.Sprintf("invalid value assigned to registration_completed")))
+				return ctx.BadRequest(jerrors)
```

Comment:
No need to do fmt.Sprintf(). No parameters here.
Also you could use errors.NewBadRequest()
And let's log this error.